provider/cmd/pulumi-resource-cloud-toolkit-aws/landingZone/accountIamArgs.ts

```diff
@@ -0,0 +1,41 @@
+export interface AccountIamArgs {
+  alias?: string;
+  passwordPolicy?: AccountPasswordPolicyArgs;
+}
+
+export interface AccountPasswordPolicyArgs {
+  enabled?: boolean;
+  rules?: AccountPasswordPolicyRulesArgs;
+  import?: boolean;
+}
+
+export interface AccountPasswordPolicyRulesArgs {
+  allowUsersToChangePassword?: boolean;
+  hardExpiry?: boolean;
+  maxPasswordAge?: number;
+  minimumPasswordLength?: number;
+  passwordReusePrevention?: number;
+  requireLowercaseCharacters?: boolean;
+  requireNumbers?: boolean;
+  requireSymbols?: boolean;
+  requireUppercaseCharacters?: boolean;
+}
+
+export const defaultPasswordPolicy = {
```

Comment:
Similar to the previous comment about exported interfaces, I would define default objects that we create just to give some context to the end user.